services/yellow/src/ambassador/YellowServiceClient.cs

```diff
@@ -37,7 +37,7 @@ public YellowServiceClient(IChannel channel)
         {
             var request = new AuthorizeOrderCancellationFeeRequest(orderId);
             var (response, exception) =
-                await _channel.Request<Response<OrderCancellationFeeAuthorizationAcquired>>(request, cancellationToken);
+                await _channel.Request<Response<bool?>>(request, cancellationToken);
```

Comment:
Use `Response<bool>` and return `response`.

Reply:
Suggestion applied